acclingo/acclingo.py

```diff
@@ -55,8 +56,9 @@ def main(self):
         
         # use individualized compare methode
         intensifier = Intensifier(tae_runner=None, stats=None, 
-                                  traj_logger=None, rng=None,
+                                  traj_logger=None, rng=RandomState(),
```

Comment:
since you use the default `Intensifier` now, you could drop this line and the `intensifier` argument for the `SMAC` constructor.